src/main/java/bio/terra/cli/command/shared/options/ControlledCloningInstructionsForUpdate.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.cli.command.shared.options;
+
+import bio.terra.workspace.model.CloningInstructionsEnum;
+import picocli.CommandLine;
+
+public class ControlledCloningInstructionsForUpdate {
+  /**
+   * Use this --cloning flog for controlled resources.
+   *
+   * <p>This class is meant to be used as a @CommandLine.Mixin.
+   */
+
+  // Can't have a default. This is used for update. Need `null` in case user does not want to
+  // update cloning instructions.
+
+  @CommandLine.Option(
+      names = "--cloning",
```

Comment:
I looked at `terra resource update bq-table`, `terra resource update bq-dataset`, etc. They all use `new` in flag:

```
      --name=<name>          Name of the resource, scoped to the workspace.
                               Only alphanumeric and underscore characters are
                               permitted.
      --new-bucket-name=<newBucketName>
                             New name of the GCS bucket, without the prefix. (e.
                               g. 'my-bucket', not 'gs://my-bucket').
      --new-name=<newName>   New name of the resource. Only alphanumeric and
                               underscore characters are permitted.
```

Can this be `--new-cloning`?

```
  private CloningInstructionsEnum newCloning;

  public CloningInstructionsEnum getNewCloning() {
    return newCloning;
  }
```

Reply:
Good find.